diff --git a/Rakefile b/Rakefile
index 8cef71a..23378f5 100644
--- a/Rakefile
+++ b/Rakefile
@@ -182,7 +182,7 @@ end
 
 # See vm.rake for more information
 desc "Build Rubinius"
-task :build => %w[build:build gems:install]
+task :build => %w[build:build]
 
 desc "Recompile all ruby system files"
 task :rebuild => %w[clean build]
diff --git a/rakelib/install.rake b/rakelib/install.rake
index b9c08ac..181e4ea 100644
--- a/rakelib/install.rake
+++ b/rakelib/install.rake
@@ -6,7 +6,7 @@
 #
 
 desc "Install Rubinius"
-task :install => %w[build:build gems:install install:files install:done]
+task :install => %w[build:build install:files install:done]
 
 # Determine if all the targets for the install directories are writable
 # decomposing each candidate directory from the right side and checking if
